src/mdx-components.tsx

```diff
@@ -1,7 +1,11 @@
 import type { MDXComponents } from "mdx/types";
+import { Body, H1, H2, H3, Subtitle } from "@/components/typography";
 
 export function useMDXComponents(components: MDXComponents): MDXComponents {
   return {
+    h1: ({ children }) => (
+      <H2 style={{ color: 'black'}}> {children} </H2>
```

Comment:
I am matching it to LG H2 as our current docs page H1 has parity with LG H2 component.